.github/ISSUE_TEMPLATE/minor-patch-release.md

```diff
@@ -1,3 +1,11 @@
+---
+name: Minor Patch Release
+about: Minor Patch Release
+title: 'Minor Patch Release Checklist'
```

Comment:
Im not 100% sure if the `'` single quotes are needed for the `title` here. I can rm them if someone can confirm they are not needed

Reply:
No worries.